04-onion-routing.md

```diff
@@ -308,10 +309,10 @@ The final node:
     - MUST add it to the HTLC set corresponding to that `payment_hash`.
     - SHOULD fail the entire HTLC set if `total_msat` is not the same for
       all HTLCs in the set.
-    - if the total `amount_msat` of this HTLC set is equal to or greater than
-      `total_msat`:
+    - if the total `amt_to_forward` of this HTLC set is equal to or greater
+      than `total_msat`:
       - SHOULD fulfill all HTLCs in the HTLC set
-    - otherwise, if the total `amount_msat` of this HTLC set is less than
+    - otherwise, if the total `amt_to_forward` of this HTLC set is less than
```

Comment:
No comments on this change, but I did find this:

https://github.com/lightning/bolts/blob/b38156b9510c0562cf50f8758a64602cc0315c19/04-onion-routing.md?plain=1#L1093

Did we forget to change that line?

Reply:
Done in https://github.com/lightning/bolts/pull/1040/commits/648fcdc0d0ce61b9acb76abdeaee19549a4d7c59